### ISSUE TITLE
It'd be cool if there were more holopads around.

### ISSUE THREAD

**Issue by Very-Soft** [VOREStation/VOREStation#1786]
TL;DR: Having played as the AI on Tether a bit, there are some areas I think should have holopads that don't.
---
One of the only ways AIs can interact with crew directly, and individually, is via the hologram. 

The bar is too far from the center holopad to reach the stools, or hear across, and there's none in the south area of the room.

There's not one in the actual AI core room itself. (There also aren't any wall radios in there to detect people talking inside the room before they breach the core. Nor one next to the core to automatically relay AI says to a channel of their choosing.)

There's no holopad or wall comms in the AI upload itself either.

There's also not one in the atmospherics or engine control rooms, which might be useful in plenty of times when comms might be down.

There's no holopads in all of Cargo. (There are some in mining though)

Holopad placement is inconsistent with the heads offices, there's none in: 
HoS office itself, there is one just outside of it. (Which kinda makes sense mostly for anti AI strats, but not under all other conditions)
RD office
CMO office
There are pads in all the other head offices.

They are also inconsistently placed in break rooms, there's none in: 
Atmos 
RnD 
Space Security 
breakrooms, where there are holopads in all the other ones that I know of.

There's none in any of the gateway rooms.

There's not one in the main area of the library. (Which is admittedly a personal favorite spot of mine to do AI RP on previous station. uwu)

There's none anywhere around the docking arms.

There's not one in Processing, which could be useful if the AI has evidence to provide. (It'd be really nice if there was a photocopier in there too for the same reason, not needed though since there are others elsewhere in sec)

Lastly, it seems to make sense to have a holopad in the lobby area of each department. Some have this, others don't. Namely Medical has one behind their desk, but not actually in the lobby. Engineering also lacks one in their lobby. It would also be nice if there was one more near to the entrance to the bridge.

I went over the station pretty thoroughly, and obviously I don't expect the AI to have holograms absolutely everywhere, but these are just some places that seem like might have been overlooked when adding the pads, or places where it might make more sense to have one.

**Comment by Fevix** [comment]
One near the center of the Holodeck would be appreciated as well.
